okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java

```diff
@@ -40,11 +40,11 @@
 
     /** Creates a metrics registry for calls from the given service to the given host. */
     public HostMetrics(TaggedMetricRegistry registry, String serviceName, String hostname) {
-        informational = registry.meter(name(serviceName, hostname, "informational"));
-        successful = registry.meter(name(serviceName, hostname, "successful"));
-        redirection = registry.meter(name(serviceName, hostname, "redirection"));
-        clientError = registry.meter(name(serviceName, hostname, "client-error"));
-        serverError = registry.meter(name(serviceName, hostname, "server-error"));
+        informational = registry.meter(name(serviceName, hostname, "1xx"));
```

Comment:
also these *xx things should be constants.

Reply:
done. left them as package private so a couple tests couldn't pick them up. will update once there's a public `MetricTags` in tritium

Reply:
https://github.com/palantir/tritium/pull/60

Reply:
Tritium 0.8.5